libs/community/langchain_community/llms/vertexai.py

```diff
@@ -288,6 +304,19 @@ def get_num_tokens(self, text: str) -> int:
 
         return result.total_tokens
 
+    def _response_to_generation(
+        self, response: TextGenerationResponse
+    ) -> GenerationChunk:
+        """Converts a stream response to a generation chunk."""
+        try:
+            generation_info = {
+                "is_blocked": response.is_blocked,
+                "safety_attributes": response.safety_attributes,
+            }
```

Comment:
Is there a class to create this instead of a dict?

Reply:
I don't know one, it looks to me it's vertex-specific.